db/setup.py

```diff
@@ -226,15 +226,24 @@
         "https://sedlabanki.is/",
         "sedlabanki.is",
         "Seðlabankinn",
-        1.0,
+        0.5,
```

Comment:
Is it a political statement to set the authority of the Central Bank to 0.5?

Reply:
Yes. For too long the force that is fiat currency has been used to oppress the masses. Only a bloody and violent anti-neoclassical economic revolution in the form of a 0.5 authority rating in our roots table can emancipate the working class!

Ehrm, no. That was supposed to be Bændablaðið.